package.json

```diff
@@ -35,7 +35,8 @@
     "neutron-html5parser": "^0.2.0",
     "node-html-parser": "^5.1.0",
     "parse5": "^6.0.1",
-    "saxes": "^6.0.0"
+    "saxes": "^6.0.0",
+    "zeed-dom": "^0.9.18"
   },
   "license": "MIT"
-}
```

Comment:
please leave the line-break :)

Reply:
Prettier in my VSCode setup added removed the trailing line. I used vim to solve the issue 😂